snap/snapcraft.yaml

```diff
@@ -41,19 +41,14 @@ apps:
     command: solana-genesis
   mint:
     command: solana-mint
-  client-demo:
-    command: solana-client-demo
```

Comment:
Are we taking this out just to put it right back in after https://github.com/solana-labs/solana/issues/504 is done?